### ISSUE TITLE
Zotero citations: ability to add a prefix or a suffix to a citation

### ISSUE THREAD

**Issue by parrenin** [ONLYOFFICE/onlyoffice.github.io#197]
In the Zotero plugin for Word or Writer, there is an option when one inserts a citation, to add a prefix or a suffix.
You can do things like: "(see Shackleton et al., 2000, for more information)".
Here the prefix is "see " and the suffix is ", for more information".
This is a useful feature when one inserts citations and I hope the OnlyOffice developers will consider it.

**Comment by pat-githubber** [comment]
@VladimirProkofiev52, maybe you could add this issue to the same enhancement 62294:
https://github.com/ONLYOFFICE/onlyoffice.github.io/issues/251#issue-1794555309

**Comment by parrenin** [comment]
Is this feature still missing?
I need this to be able to switch to OnlyOffice for writing my scientific articles.

**Comment by pat-githubber** [comment]
Yes, I just checked it. Adding a pre- or suffix is not possible ☹️

**Comment by VladimirProkofiev52** [comment]
Hello! @parrenin 
Recently, we released plugin version 1.0.7, which includes the functionality you requested. Please update the plugin to the latest version to see these changes.

Thank you for your report!

**Comment by parrenin** [comment]
Thank you @VladimirProkofiev52 !
I tested this new functionality and it works well, much appreciated!
Have a nice evening.
